firefox-ios/Extensions/ShareTo/ShareViewController.swift

```diff
@@ -106,6 +106,7 @@ class ShareViewController: UIViewController {
         }
 
         let profile = BrowserProfile(localName: "profile")
+        Viaduct.shared.useReqwestBackend()
```

Comment:
This is the one line to make the network request go ahead, but we need the rest otherwise we'd get a pretty bad flickering experience